cmd/pr-reminder/main.go

```diff
@@ -283,10 +286,16 @@ func main() {
 		logrus.WithError(err).Fatal("failed to load config")
 	}
 
-	var gtk githubToKerberos
-	if err = loadConfig(o.githubMappingConfig, &gtk); err != nil {
+	roverUsers := []*rover.User{}
+	if err = loadConfig(o.githubUsers, &roverUsers); err != nil {
 		logrus.WithError(err).Fatal("failed to load rover groups config")
 	}
+	var gtk githubToKerberos = rover2github.MapGithubToKerberos(roverUsers,
```

Comment:
I would define the function in https://github.com/openshift/ci-tools/blob/master/pkg/rover/types.go
and avoid the new package.

The conflict function as an arg looks great but I would just push it into the `MapGithubToKerberos`.
I cannot see other usage as now. I could do it when we have another use case.

We can reuse the same function [here](https://github.com/openshift/ci-tools/blob/master/cmd/github-ldap-user-group-creator/main.go#L230) as well.

Reply:
Changed as you requested and just opened also [#3418](https://github.com/openshift/ci-tools/pull/3418)